compose/foundation/foundation/src/uikitMain/kotlin/androidx/compose/foundation/Magnifier.uikit.kt

```diff
@@ -213,11 +213,11 @@ internal class MagnifierNode(
     private var previousSize: IntSize? = null
 
     fun update(
-        sourceCenter: Density.() -> Offset,
-        magnifierCenter: (Density.() -> Offset)?,
-        onSizeChanged: ((DpSize) -> Unit)?,
-        color: Color,
-        platformMagnifierFactory: PlatformMagnifierFactory
+        sourceCenter: Density.() -> Offset = this.sourceCenter,
+        magnifierCenter: (Density.() -> Offset)? = this.magnifierCenter,
+        onSizeChanged: ((DpSize) -> Unit)? = this.onSizeChanged,
+        color: Color = this.color,
```

Comment:
Do we need these default arguments in current PR? If no, I would revert them.

Reply:
The only property that has to be updated here is [color ](https://github.com/JetBrains/compose-multiplatform-core/pull/1549/files#diff-1f10431f7e006c362fa801b215036af03b87d52426d1f4cba041e0c4314fbb18R114)so i added default parameters to make it more readable